modules/library/common-search/src/test/java/org/fao/geonet/common/search/processor/impl/RssResponseProcessorImplTest.java

```diff
@@ -120,6 +141,8 @@ public static PropertySourcesPlaceholderConfigurer properties() throws Exception
 			properties.setProperty("gn.site.organization", "momorg");
 			properties.setProperty("gn.baseurl", "http://gn5:8277/geonetwork");
 			properties.setProperty("gn.linkToLegacyGN4", "true");
+			properties.setProperty("gn.linkToCustomMetadataUrl", "false");
+      properties.setProperty("gn.customMetadataUrl", "http://gn:8277/custom");
```

Comment:
looks like the file is using tabs and your changes are using spaces... ah, the good old times ;)